game/core/src/me/lihq/game/models/Room.java

```diff
@@ -113,21 +130,73 @@ public void moveClue(Clue clue, int x, int y)
      */
     public void addClue(Clue newClue)
     {
+        System.out.println("Added Clue " + newClue.getName() + " at location " + newClue.getPosition() + " in room " + getID());
+
         if (!cluesInRoom.contains(newClue)) {
             cluesInRoom.add(newClue);
         }
     }
 
+
     /**
-     * This removes a clue from the room
+     * This method takes a location parameter and checks it for a clue, if a clue is found it is removed from the map and retunr
```

Comment:
retunr, nice. Ill go through all these comments you're making now later and fix them all!